tests/ext/argparse_tests.py

```diff
@@ -474,6 +478,19 @@ def test_alternative_default(self):
             self.eq(res,
                     "Inside AlternativeDefault.alternative_default")
 
+    def test_default_default(self):
+        self.reset_backend()
+        self.app = self.make_app(APP,
+                                 argv=['default_default'],
+                                 argument_handler=ArgparseArgumentHandler,
+                                 handlers=[
+                                     NoDefaultFunction,
+                                 ],
+                                 )
+        with self.app as app:
+            app.run()
+            # Expectation: No exception
```

Comment:
Perhaps we could check what the function actually outputs, but this is all I could figure out in a short time.